main/zoe/guide/contracts/callSpread.md

```diff
@@ -0,0 +1,77 @@
+# Call Spread
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/master/packages/zoe/src/contracts/callSpread.js) (Last updated: 03-NOV-2020)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+This contract implements a fully collateralized call spread. A call spread can be used as a
+financial building block to create futures, puts, calls, and event binaries that would form the
+basis for a prediction market, insurance, and much more. A Call Spread is a combination of a call
+option bought at one strike price and a second call option sold at a higher price. That means there
+are two parties to the call spread, each holding an ERTP asset that pays out complementary amounts
+depending on the value of some good at a known future time.
+
+The Zoe invitations representing the options are produced in pairs that represent the matching
+assets.  The purchaser deposits the entire amount that will be approtioned between the holders of
+the two positions. The individual options are ERTP invitations that are suitable for resale.
+
+This option is settled financially. There is no requirement that the original purchaser have
+ownership of the underlying asset at the start, and the beneficiaries shouldn't expect to take
+delivery at closing.
+
+It is common for the strike price and collateral to use the same currency, though this contract
+decouples them. This would be like a spread based on the price of APPL stock (`Underlying`),
+where the stock price is measured in USD (`Strike`), but the contract pays out in JPY
+(`Collateral`).
+
+The issuerKeywordRecord specifies the issuers for four keywords: Underlying, Strike, Collateral,
+and Quote. The original deposit and the payout are in Collateral. Strike amounts are used for the
+price oracle's quote as to the value of the Underlying, as well as the strike prices in the
+terms. The priceAuthority's quotes are ERTP assets issued by a QuoteAuthority, which must be
+specified using the keyword Quote.
+
+The terms include { timer, underlyingAmount, expiration, priceAuthority, strikePrice1,
+strikePrice2, settlementAmount }. `timer` must be recognized by `priceAuthority`. `expiration` is a
+time recognized by the `timer`. `underlyingAmount` is passed to `priceAuthority`, so it could be an
+NFT or a fungible amount. `strikePrice2` must be greater than `strikePrice1`. `settlementAmount`
+uses Collateral.
```

Comment:
```suggestion
The terms include:
- `timer`: Must be recognized by `priceAuthority`.
- `underlyingAmount`: Passed to `priceAuthority`, so it could be an NFT or a fungible amount. 
- `expiration`: A time recognized by the `timer`. 
- `priceAuthority`
- `strikePrice1`: Must be less than `strikePrice2`.
- `strikePrice2`: Must be greater than `strikePrice1`.
- `settlementAmount`:  Uses Collateral.
```

Reply:
Please add something for priceAuthority and expand on how settlementAmount uses Collateral.